scss/_patterns_search-and-filter.scss

```diff
@@ -6,10 +6,14 @@
 }
 
 @mixin vf-search-and-filter {
-  .p-search-and-filter {
-    $height: 2.5rem;
+  $search-filter-actions-width: map-get($icon-sizes, default) + $sph-inner * 2;
```

Comment:
Can we have a short comment what it defines and how it's calculated?

Reply:
you can copy mine from the notifications. Same calculation